lib/hsa/mcwamp_hsa.cpp

```diff
@@ -1749,9 +1735,8 @@ class HSAQueue final : public KalmarQueue
         // do read
         if (dst != device) {
             if (!getDev()->is_unified()) {
-#if KALMAR_DEBUG
-                std::cerr << "read(" << device << "," << dst << "," << count << "," << offset << "): use HSA memory copy\n";
-#endif
+                DBOUT(DB_COPY, "read(" << device << "," << dst << "," << count << "," << offset 
```

Comment:
Should we use DBOUTL instead of adding the \n to end of DBOUT for these outputs?

Reply:
I won't change existing debug messages.  In fact I wouldn't consider DBOUTL a necessity in the first place